nni/nas/space/frozen.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+__all__ = ['current_model', 'model_context']
+
+import copy
+from typing import Optional, Dict, Any
+from nni.mutable import frozen_context, ContextStack
+
+
+def current_model() -> Optional[dict]:
+    """Get the current architecture dict in :func:`model_context`.
```

Comment:
Is there anywhere explaining what's the difference between architecture and model?

Reply:
Architecture is the term used in v2.x. I'm trying to use "model" to refer to "architecture" everywhere.